idaes/generic_models/unit_models/pressure_changer.py

```diff
@@ -455,6 +479,38 @@ def actual_work(b, t):
                     b.work_isentropic[t] * b.efficiency_isentropic[t]
                 )
 
+        # Add head expressions (often calcuated from performance curve)
+        try:
+            @self.Expression(self.flowsheet().config.time)
+            def head_isentropic(b, t): # units are energy/mass e.g. J/kg
```

Comment:
We should not be presuming set units here, as the PressureChanger gets it's units from the property package. We probably need to make use of unit conversions for this.

Reply:
I'm not assuming anything J/kg are just and example in the comment not necessarily the expected unit.  Whatever the property package uses is what it is.  I'll fix the comment to be clear.